packages/react-magma-dom/src/theme/magma.ts

```diff
@@ -575,43 +579,44 @@ export const magma = {
   },
   typographyExpressiveVisualStyles: {
     heading2XLarge: {
-      mobile: typeScale.size13,
+      mobile: typeScale.size11,
       desktop: typeScale.size15,
-      fontWeight: 600,
+      fontWeight: 500,
     },
     headingXLarge: {
-      mobile: typeScale.size10,
-      desktop: typeScale.size12,
-      fontWeight: 400,
+      mobile: typeScale.size09,
+      desktop: typeScale.size11,
+      fontWeight: 600,
+      lineHeight: typeScale.size13,
```

Comment:
Mmmm, this feels wrong. The lineHeight needs to match the typeScale. So `headingXLarge`'s desktop lineHeight needs to be whatever we set `typeScale.size11.lineHeight`. Lets work with Orion to make sure this is the case

Reply:
`size11` now line-heights a whole `64px`!